source/security/index.md

```diff
@@ -174,6 +174,31 @@ or define an error page
 </global-exception-mappings>
 ```
 
+### Proactively protect from OGNL Expression Injections attacks if easily applicable
+
+The framework has a history of critical security bugs, many tied to its use of OGNL technology; Due to its ability to 
+create or change executable code, OGNL is capable of introducing critical security flaws to any framework that uses it. 
+Multiple Struts 2 versions have been vulnerable to OGNL security flaws. Consequently, we've equipped OGNL and the 
+framework with following proactive optional possibilities since OGNL 3.1.24 and Struts 2.5.22. They're disabled by 
+default but via enabling them, you can proactively protect from potential still unknown OGNL Expression Injections flaws:
+
+NOTE: These might break your current app functionality. Before using in production environment, you're recommended to 
+comprehensively test your app UI and functionalities with these enabled.
```

Comment:
Use `>` to make a note

```
> **NOTE**: These might break your current app functionality. Before using in production environment, you're recommended to 
> comprehensively test your app UI and functionalities with these enabled.
```

Reply:
Thanks 👍 - I was remembering there is a template and had searched to find it but couldn't ;)